src/raw_node.rs

```diff
@@ -220,46 +216,15 @@ pub struct RawNode<T: Storage> {
 impl<T: Storage> RawNode<T> {
     #[allow(clippy::new_ret_no_self)]
     /// Create a new RawNode given some [`Config`](../struct.Config.html) and a list of [`Peer`](raw_node/struct.Peer.html)s.
-    pub fn new(config: &Config, store: T, mut peers: Vec<Peer>) -> Result<RawNode<T>> {
+    pub fn new(config: &Config, store: T) -> Result<RawNode<T>> {
```

Comment:
This is a break in public API (and a big one) so we'll need to make this a `0.6.0` release.

Reply:
I agree if you don't oppose to remove the parameter.

Reply:
I support removing it if it's only meant for testing.